extensions/wikia/Wall/WallHelper.class.php

```diff
@@ -582,23 +579,41 @@ public static function getWallTitleData( $rc = null, $row = null ) {
 		if ( !( $objTitle instanceof Title ) ) {
 			// it can be media wiki deletion of an article -- we ignore them
 			Wikia::log( __METHOD__, false, "WALL_NOTITLE_FOR_MSG_OPTS " . print_r( [ $rc, $row ], true ) );
-			wfProfileOut( __METHOD__ );
 			return true;
 		}
 
+		// SUS-1777: Don't bother trying to load Wall Message if this is the Wall itself
+		if ( $row->page_namespace === NS_USER_WALL ) {
+			// change in NS_USER_WALL namespace mean that wall page was created (bugid:95249)
+			$title = Title::makeTitle( NS_USER_WALL, $row->page_title );
+
+			$out = [
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$out". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)